pkg/clusteragent/orchestrator/controller.go

```diff
@@ -33,6 +33,14 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+const (
+	payloadTypePod        = "pod"
+	payloadTypeDeployment = "deployment"
```

Comment:
If both `payloadTypeDeployment` and `forwarder.PayloadTypeDeployment` must have the same value, one of them should be removed.

Reply:
I will remove the latter.